domains/madkarma.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "madkarmaa",
+        "email": "madkarma99@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "madkarma"
```

Comment:
Invalid CNAME, must be a valid hostname.

Reply:
should be correct